packages/components/src/playground/button.spec.tsx

```diff
@@ -8,12 +8,12 @@ describe("<Button />", () => {
     expect(container.firstChild).toMatchInlineSnapshot(`
       <div>
         <button
-          class="czedi-btn czedi-btn-primary"
+          class="sc-bdfBwQ sc-gsTCUz fvVoTP bMZeDH"
```

Comment:
to get unchanging output here, we can use https://github.com/chanzuckerberg/along/blob/cc2a3feac8ab7464014126bf3b1301a88b327d47/client/src/test/removeStylesJestSerializer.ts, though we can definitely wait until another PR for that.

Reply:
ooh cool, yeah I want to show the className change once we add twin.macro but this is good to consider after that